platform-ajax/src/main/java/com/softicar/platform/ajax/engine/AjaxDomEngine.java

```diff
@@ -489,6 +489,12 @@ public AjaxResourceLinkRegistry getResourceLinkRegistry() {
 
 	// -------------------------------- browser manipulation -------------------------------- //
 
+	@Override
+	public void updateDocumentTitle(String pageTitle) {
```

Comment:
why not `setDocumentTitle(IDisplayString)`? wouldn't that be better

Reply:
dunno, changed to set